3-Authorization-II/2-call-api-b2c/AppCreationScripts/sample.json

```diff
@@ -1,82 +1,81 @@
 {
     "Sample": {
-      "Title": "A React single-page application using MSAL React to authorize users for calling a protected web API on Azure AD B2C",
-      "Level": 200,
-      "Client": "React SPA",
-      "Service": "Node.js web API",
-      "RepositoryUrl": "ms-identity-javascript-react-tutorial",
-      "Endpoint": "AAD v2.0",
-      "Provider": "B2C"
+        "Author": "Salman",
+        "Title": "A React single-page application using MSAL React to authorize users for calling a protected web API on Azure AD B2C",
+        "Level": 200,
+        "Client": "React SPA",
+        "Service": "Node.js web API",
+        "RepositoryUrl": "ms-identity-javascript-react-tutorial",
+        "Endpoint": "AAD v2.0",
+        "Description": "A React single-page application using MSAL React to authorize users for calling a protected web API on Azure AD B2C",
+        "Languages": ["javascript", "nodejs"],
+        "Products": ["React", "Express", "azure-active-directory", "msal-js", "passport-azure-ad"],
```

Comment:
> "Products": ["React", "Express", "azure-active-directory", "msal-js", "passport-azure-ad"],

React and Express are not microsoft products. In general, double check what you put here matches the product taxonomy [here](https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main). Same goes for `Languages`

Reply:
This is not resolved, but please do not set to resolved without addressing